.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule ".github/actions/get-workflow-origin"]
+	path = .github/actions/get-workflow-origin
+	url = https://github.com/potiuk/get-workflow-origin
```

Comment:
BTW. This change should use SHA not master version otherwise we (implicitly) fall in the same trap as https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/security-hardening-for-github-actions#using-third-party-actions 

This is why subrepo is also much nicer because you always have specific commit of the repo you are linking to. You cannot link to branch - you bring very specific version of the code in.

Reply:
It doesn't -- that is now how submodules work. It has a pinned sha, just not in this file -- it's a "property" of the submodule path itself.

![image](https://user-images.githubusercontent.com/34150/103776055-cfc97d80-5026-11eb-827a-01e85a5bc33e.png)

Reply:
I see- so it is stored in the git tree https://stackoverflow.com/questions/5033441/where-does-git-store-the-sha1-of-the-commit-for-a-submodule . That's fine. So then it's great that GitHub shows the actual files in review then. otherwise it woudl have been even more difficult to review :)